### ISSUE TITLE
Inviting users doesn't assign role correctly.

### ISSUE THREAD

**Issue by innominata** [YABhq/Laracogs#12]
The generated view for inviting users :

`@input_maker_create('roles', ['type' => 'relationship', 'model' => 'App\Repositories\Role\Role', 'label' => 'label'])`

Is returning the integer Id of the role dropdown (normal behaviour iirc), which UserService.php 
```
public function create($user, $password, $role = 'member', $sendEmail = true)
    {
        try {
            DB::beginTransaction();
                // create the user meta
                $this->userMetaRepo->findByUserId($user->id);
                // Set the user's role
                $this->userRepo->assignRole($role, $user->id);

```
is passing to UserRepository.php which is using a method 'findByName'  on Repositories\Role.php. 

```
public function assignRole($roleName, $userId)
    {
       
        $role = Role::findByName($roleName);
        $user = $this->model->find($userId);

        $user->roles()->attach($role);
    }
```

This causes the user to not be assigned a role.

**Comment by mlantz** [comment]
Look at the docs:

@input_maker_create('roles', ['type' => 'relationship', 'model' => 'App\Repositories\Role\Role', 'label' => 'label', 'value' => 'name']) 

you can set the label and value :)

**Comment by mlantz** [comment]
Corrected in 1.9.12
